docs/source/related_topics/technical/client-support-grid.md

```diff
@@ -0,0 +1,86 @@
+```python
```

Comment:
fixed a bunch of inaccuracies in the support grid. reorg'd clients. added column for OS.